mailpoet/assets/js/src/automation/components/option-button/index.tsx

```diff
@@ -38,7 +38,7 @@ export function OptionButton({
       </Button>
       {slots.length > 0 &&
         slots.map(({ key, slot }) => (
-          <Fragment key={`slot-${key}`}>{slot}</Fragment>
+          <Fragment key={`slot-${key}`}>{slot()}</Fragment>
```

Comment:
Oh, did this work before? I also wonder if there's a difference between `{slot()}` and `<Slot>`...